x-pack/plugins/uptime/public/components/monitor/status_details/location_map/embeddables/embedded_map.tsx

```diff
@@ -26,7 +26,7 @@ import {
 import { MAP_SAVED_OBJECT_TYPE } from '../../../../../../../maps/public';
 import { MapToolTipComponent } from './map_tool_tip';
 // eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { RenderTooltipContentParams } from '../../../../../../../maps/public/classes/tooltips/tooltip_property';
+import type { RenderTooltipContentParams } from '../../../../../../../maps/public/classes/tooltips/tooltip_property';
```

Comment:
can we please export this type from map? after that we can remove `// eslint-disable-next-line @kbn/eslint/no-restricted-paths`